README.md

```diff
@@ -911,16 +917,18 @@ It is also necessary to fill out the `google` stanza in the configuration file.
 
 ```hocon
 google {
-  authScheme = "service"
+  applicationName = "cromwell"
+  
+  cromwellAuthenticationScheme = "service_account"
 
-  // If authScheme is "user"
+  // If cromwellAuthenticationScheme is "user_account"
```

Comment:
The idea was to make a clear distinction between the `userAuthenticationScheme` which helps authenticating a user (per workflow), and the `cromwellAuthenticationScheme` which is cromwell-wide.
But maybe `authenticationScheme` is clear enough ?